test/unit/import_playbooks_test.rb

```diff
@@ -0,0 +1,74 @@
+require 'test_plugin_helper'
+require_relative '../../app/services/foreman_ansible/playbooks_importer'
+require_relative '../fixtures/output_example_from_playbooks_importer'
```

Comment:
this should work:
```suggestion
require_dependency 'foreman_ansible/playbooks_importer' # IMO omitting this would also work as it should get auto required :)
require 'fixtures/output_example_from_playbooks_importer'
```